### ISSUE TITLE
Provide Plug support

### ISSUE THREAD

**Issue by alex-shamshurin** [hhsnopek/vim-sugarss#3]

**Comment by hhsnopek** [comment]
Hey @alex-shamshurin, are you referring to [junegunn/vim-plug](https://github.com/junegunn/vim-plug)?

**Comment by alex-shamshurin** [comment]
Yes! I'd like install via Plug like other plugins

**Comment by hhsnopek** [comment]
I'm actually using that right now to install it! I left it out of the readme because I didn't want to list _all_ plugin managers for vim 😅 

Here's my snippet: https://github.com/hhsnopek/dotfiles/blob/master/.nvimrc#L32
```viml
" Generic vim-plug usage (not my snippet)
Plug 'hhsnopek/vim-sugarss'
```

**Comment by alex-shamshurin** [comment]
Of course, but it do not include `autoload` dir or `plugin` dir and I thought it's not for Plug

**Comment by hhsnopek** [comment]
Oh, it contains a `ftdetect` directory that'll resolve the files - is it not loading for you?

**Comment by alex-shamshurin** [comment]
It hangs on 'Cloning git...' via PlugInstall, but `git clone` `works fine via console.
![sugarss_plugin](https://user-images.githubusercontent.com/1595209/29703405-2638e5fc-897e-11e7-8eab-256775356b4a.jpg)

**Comment by cseelus** [comment]
Just tried it, works (NVIM v0.2.2, macOS 10.13.1)

**Comment by alex-shamshurin** [comment]
This is because of Windows cmd. Plug cannot get status of git operation whether it done or not
